vm/error.go

```diff
@@ -22,7 +22,6 @@ type Error struct {
 	message      string
 	stackTraces  []string
 	storedTraces bool
-	Type         string
```

Comment:
The `Type` attribute is essentially the error's class name. So we don't need to store it separately.